inventory_reporting/reports/inventory_report.py

```diff
@@ -135,6 +142,26 @@ def generate_storable_report(self, workbook, wizard):
                         "ilike",
                         "Virtual Locations/Inventory adjustment",
                     ),
+                    (
+                        "stock_move_id.picking_id",
+                        "=",
+                        False,
+                    ),
+                ],
+            },
+            {
+                "name": _("Scrap"),
+                "filter": [
+                    (
+                        "stock_move_id.scrapped",
+                        "!=",
+                        False,
+                    ),
+                    (
+                        "stock_move_id.picking_code",
+                        "!=",
+                        "internal",
+                    ),
```

Comment:
@yostashiro san, I just reconsider it and seems this is not needed in the client case. I was thinking that if the user proceed the scrap directly from the receipt transfer, the receipt transfer data will shown both in receipt sheet and scrap sheet. but since our client use the Scrap operation type to record the scrap, we can delete this lines.

By the way, since there's several request to add new sheets, as you mentioned in the past @yostashiro san, I think the design should be improved to be not hardcode the filter each sheets. maybe it can be changed to have 1 menu to provide setting form to add and define only the sheets of storable and consumable reports. for inventory valuation seems its ok with current design. how do you think @AungKoKoLin1997